capstone/src/main/java/com/step902020/capstone/Event.java

```diff
@@ -15,7 +15,6 @@
   @Id
   Long datastoreID;
 
-  @Reference
```

Comment:
Could you create a follow up issue to resolve the issue this was causing?

Reply:
We came up with a solution to this problem. For the event side all the information that it needs from the organization is the organization id and the organization name. The plan is to directly store these as fields instead of as a reference and once we create the event element it will have a link to the public organization page with more info. A PR is coming soon to address this issue.

Reply:
Sounds good. It's a good example of the kind of architectural tradeoffs involved in building software!